apps/tenant-management-webapp/src/app/pages/admin/dashboard/dashboard.tsx

```diff
@@ -245,6 +244,10 @@ const Dashboard = (): JSX.Element => {
                         represent a service that is useful to the end user by itself, such as child care subsidy and
                         child care certification.
                       </div>
+                      <div>&nbsp;</div>
```

Comment:
Using  a <span /> with style might be nicer here.

Reply:
will be address later.